gapidapk/android/apk/rules.bzl

```diff
@@ -102,7 +102,11 @@ def gapid_apk(name = "", abi = "", pkg = "", libs = {}, bins = {}):
         srcs = select({
             "//tools/build:android-" + name: [name + "_libs"],
             "//conditions:default": [],
-        })
+        }),
+        deps = select({
+            "//tools/build:android-" + name: ["@ndk_vk_validation_layer//:" + abi],
+            "//conditions:default": [],
+        }),
```

Comment:
does it not work if you just make it a dep below in the android_binary target?

Reply:
I tried this in native.android_binary:

```
        deps = [
            "//gapidapk/android/app/src/main:gapid",
            ":" + name + "_native",
        ] + select({
            "//tools/build:android-" + name: ["@ndk_vk_validation_layer//:" + abi],
        },
	no_match_error = "ERROR"
	),
```

This does hit the error, I'm not sure why we cannot rely on the same condition as in native.android_library?

Another option is:

```
        deps = [
            "//gapidapk/android/app/src/main:gapid",
            ":" + name + "_native",
            "@ndk_vk_validation_layer//:" + abi
        ],
```

But this leads to all APKs embedding validation layers for all ABIs.